src/components/Routes/ClientRoutes.js

```diff
@@ -1,15 +1,29 @@
-import React from "react";
-import { Routes, Route } from "react-router-dom";
-import Wallet from "../../pages/Wallet/Wallet";
-import TransferStatus from "../../pages/TransferStatus/TransferStatus";
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import SendTokens from '../../pages/SendTokens/SendTokens';
+import TransferStatus from '../../pages/TransferStatus/TransferStatus';
+import Wallet from '../../pages/Wallet/Wallet';
 
 const ClientRoutes = () => {
-  return (
-    <Routes>
-      <Route path="/" exact element={<Wallet />} />
-      <Route path="/transfer-status" exact element={<TransferStatus />} />
-    </Routes>
-  );

```

Comment:
Also, if you use Visual Studio Code, please enable 'Prettier' as a code formatter for this project so we all have the same code layout.

Reply:
Sorry, I was using custom settings for my Prettier. I have changed the settings to match this project now.

Reply:
No problem, thank you for changing. I will add this to the README later.